src/github.rs

```diff
@@ -1,396 +1,421 @@
-use serde::{Deserialize, Serialize};
+use serde::{
```

Comment:
This file is using tabs over spaces, shouldn't we be using 4 spaces?

Reply:
Would prefer tabs since that is more or less Parity style.  Have included my `.rustfmt.toml`. I use `rustfmt` with `hard_tabs = true`, meaning tabs for indentation, spaces for "alignment", which is the closest config I could find.